lib/getMergedPullRequests.js

```diff
@@ -21,21 +21,14 @@ function getPullRequests(fromTag) {
 
             return mergeCommits
                 .map((commit) => commit.match(/^Merge pull request #(\d+) from (.*?) \((.*)\)$/u))
-                .filter((matches) => matches)
+                .filter((matches) => matches && parseInt(matches[1], 10) > 500)
```

Comment:
LOL commented in the commit log, but should have commented in the code. Anyway, it's temporary and very Storybook 3.0 specific. 😭